src/utils/hints/contractsHints.js

```diff
@@ -3,4 +3,17 @@ export const contractsHints = {
   contractCreated: 'Keyblock height and exact date and time when the smart contract was created.',
   contractTxHash: 'The transaction hash of a ContractCreateTx that was executed to create the smart contract on the blockchain.',
   contractCreator: 'Account that created the smart contract.',
+  bytecode: 'Pre-compiled code of the Smart Contract.',
+  contractsAccount: 'The address where the Smart Contract is located. To be used if a Smart Contract is already deployed.',
+  contractsAccountBalance: 'Smart Contract account balance. Balance is needed to execute "payable" functions.',
+  contractCalls: 'Amount of calls to the Smart Contract.',
+  apiLinks: 'Links to sources of the Smart Contract data.',
+  callTransactionsHash: 'Unique identifier of ContractCallTx which represents call to Smart Contract function.',
+  callTransactionsTime: 'Estimated date, time and block height when the Smart Contract was called.',
+  callTransactionsCaller: 'Account that called the Smart Contract.',
+  callTransactionsEntryPoint: 'Name of the Smart Contract function that was called.',
+  eventsCallTransaction: 'Unique identifier of ContractCallTx which emitted the event.',
+  eventsCreated: 'Estimated date, time and block height when the Smart Contract was emitted the event.',
```

Comment:
Did you mean "(...) when the Smart Contract had the event  emitted."?

Reply:
accepted with a bit of rephrasal

Reply:
Thanks, even better!